contributing/code_of_conduct/reporting_guidelines.rst

```diff
@@ -0,0 +1,99 @@
+Reporting Guidelines
+====================
+
+If you believe someone is violating the Code of Conduct we ask that you report
+it to the :doc:`enforcement team </contributing/code_of_conduct/enforcement_team>`
+by emailing, twitter, in person or any way you see fit.
+
+**All reports will be kept confidential.** The privacy of everyone included in
+the report is of our highest concern. Second to privacy there is transparency.
+After every report we will determine if a public statement should be made. If
+that's the case, the identities of all victims, reporters, and the accused will
+remain confidential unless those individuals instruct us otherwise. The details
+of the incident may also be generalized.
+
+If you believe anyone is in physical danger, please notify appropriate emergency
```

Comment:
What about something more generic. If you think someone did something that is against the law (not just because one is in physical danger), legal authorities should be contacted first (as that cannot be our responsibility to do so).

Reply:
I agree, I would just drop this whole paragraph, it's common sense, and not the role of a software project, to contact the appropriate emergency services in case of danger.

I would not speak about "something against the law" either, because it can mean things very different depending of the country of residence of the person (as I pointed out before, having homosexual relations is illegal in many countries, and I guess we don't want such people being reported).